packages/agent/package.json

```diff
@@ -46,17 +45,19 @@
   ],
   "exports": {
     ".": {
-      "import": "./dist/esm/main.js",
-      "require": "./dist/cjs/main.js",
-      "types": "./dist/types/main.d.ts"
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js",
+      "types": "./dist/types/index.d.ts"
     }
   },
-  "react-native": "./dist/esm/main.js",
+  "react-native": "./dist/esm/index.mjs",
```

Comment:
Having issues with this on React Native, as there's not an `index.mjs` file in the dist/esm folder. Should this be `index.js`?

Reply:
aye i think this should be `index.js`.

Reply:
good catch -- thanks @amika-sq 
typo on my part

Addressed in: https://github.com/TBD54566975/web5-js/pull/168/commits/d662236d5c2ad6c86177f729faf9a5eb180f6aa0